### PR TITLE
close #150 by safely rendering HTML descriptions with dompurify

### LINKED ISSUE #150: safely render html descriptions
Sometimes descriptions contain simple HTML such as links (see example below). We can actually the description string as an html element. Typically this can be a security vulnerability if handled incorrectly.

**Read https://reactjs.org/docs/dom-elements.html#dangerouslysetinnerhtml for more on this**. Also read https://medium.com/better-programming/what-is-dangerouslysetinnerhtml-6d6a98cbc187 to see how to implement this

Screen Shot 2020-11-08 at 1 48 47 PM

### PR DESCRIPTION
**Pull request by hamirmahal**
This pull request should close #150 if we merge it. This pull request uses dompurify to sanitize HTML from Freesound before rendering that HTML in this project on a sound's soundpage.﻿

### CODE REVIEW

File: tsconfig.json [hunk 1]
```diff
@@ -0,0 +1,26 @@
+{
```
Comment (hamirmahal) [inline tsconfig.json:1]: I'm honestly not sure why `tsconfig.json` changed. Did `dompurify` update it after I installed it for this repository?

Comment (amilajack) [reply tsconfig.json:1]: Try rebasing against master. This should fix this. `tsconfig.json` was removed recently:

```
git pull --all
git rebase master
```

**Review by amilajack — CHANGES_REQUESTED** [review]

**Comment by hamirmahal** [comment]
https://freesound-player-git-render-html.amilajack.vercel.app/sound/464889 is an example of a sound with a description that looks more elegant after the changes made in this pull request, i.e., installing and using the `dompurify` package to sanitize HTML from Freesound before rendering that HTML in the description section of a sound's page.

Screen Shot 2020-11-09 at 3 18 11 PM

File: tsconfig.json (hunk 1 above)
Comment (hamirmahal) [reply tsconfig.json:1]: It doesn't look like my local files changed. I also think I fetched and pulled the latest changes before starting work on this new branch, but I might be wrong.

Screen Shot 2020-11-09 at 3 19 54 PM

Comment (hamirmahal) [reply tsconfig.json:1]: It looks like we removed `tsconfig.json` in 417337d519d1adea5d1fb47789bec385b5116486 approximately three hours ago from the time of this writing.

Comment (amilajack) [reply tsconfig.json:1]: There is no `tsconfig.json` in master branch. See https://github.com/amilajack/freesound-player

Comment (hamirmahal) [reply tsconfig.json:1]: It looks like the changes from this pull request are "on top of" the latest changes, including commit 417337d519d1adea5d1fb47789bec385b5116486 where we deleted `tsconfig.json`. I'm not sure why `tsconfig.json` is back, but should I just remove it and push that change here?
 
Screen Shot 2020-11-09 at 3 42 59 PM

**Review by amilajack — APPROVED** [review]
🥳 🥳 🥳

**Comment by amilajack** [comment]
@hamirmahal nice job! Feel free to merge

**Comment by amilajack** [comment]
merging to prevent merge conflicts

**Pull request merged**